src/frontend/src/handler/create_table.rs

```diff
@@ -656,6 +664,7 @@ fn gen_table_plan_inner(
             .collect_vec(),
         pk_column_ids: pk_column_ids.iter().map(Into::into).collect_vec(),
         properties: with_options.into_inner().into_iter().collect(),
+        options: options.unwrap(),
```

Comment:
The `unwrap` here always succeeds because it is inside `source_info.map`. In other words, existence of `source_info` guarantees existence of `options`. Shall we include `options` as part of `source_info`? Note that enum `format` and `row_encode` are also part of `source_info`.

Reply:
> > In other words, existence of source_info guarantees existence of options
> 
> I don't quite get how you get the conclusion.

If this claim is not true in some cases, the `unwrap` would panic and should be replaced by proper error handling.

Reply:
>  Shall we include options as part of source_info?

Done.